core/Cargo.toml

```diff
@@ -66,8 +66,9 @@ derive_more = "0.99.17"
 getset = "0.1.2"
 itertools = "0.10.3"
 
-[dev-dependencies]
 tempfile = "3.3.0"
```

Comment:
We need `tempfile` on a regular basis now?

Reply:
Yes because tests in other crates need it to be present.